registry/handlers/app.go

```diff
@@ -641,7 +654,7 @@ func (app *App) context(w http.ResponseWriter, r *http.Request) *Context {
 	context := &Context{
 		App:        app,
 		Context:    ctx,
-		urlBuilder: v2.NewURLBuilderFromRequest(r),
```

Comment:
`app.httpHost` is a `url.URL`. If it's indeed unnecessary to parse the path from the request in the first case, this could be:

    var ub *v2.URLBuilder
    if app.httpHost != "" {
        ub = v2.NewURLBuilder(&app.httpHost)
    } else {
        ub =  v2.NewURLBuilderFromRequest(r)
   }